content/05.main-text.md

```diff
@@ -1,9 +1,60 @@
 ## Introduction {.page_break_before}
 
+A *network* (also known as a [graph](https://en.wikipedia.org/wiki/Graph_(discrete_mathematics))) is a conceptual representation of a group of things — called *nodes* — and the relationships between them — called *edges*.
+Typically, a network has only one type of node and one type of edge.
+But in many cases, it is necessary to be able to distinguish between different types of entities and relationships.
+
+### Hetnets
+
+A *hetnet* (short for **het**erogeneous information **net**work [@doi:10.15363/thinklab.d104]) is a network where nodes and edges can be multiple types.
+This additional dimension allows a hetnet to accurately describe more complex data.
```

Comment:
I don't love the word "dimension" here. What do you think about representational flexibility? Or something else...

Reply:
done in https://github.com/greenelab/connectivity-search-manuscript/pull/5/commits/cc608e695b788595883a89f12f389e905977020d

this was from the website and was too general for the manuscript so I tweaked all of these sentences.